test/e2e/gke_local_ssd.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-var _ = framework.KubeDescribe("GKE local SSD [Feature:GKELocalSSD]", func() {
+var _ = framework.SIGDescribe("storage", "GKE local SSD [Feature:GKELocalSSD]", func() {
```

Comment:
Please move this file to `test/e2e/storage` and add `k8s.io/kubernetes/test/e2e/storage/utils` to imports
```suggestion
var _ = utils.SIGDescribe("GKE local SSD [Feature:GKELocalSSD]", func() {
```

Reply:
thanks @spiffxp , PTAL and see if the re-arrangement is as suggested (I tried to follow https://github.com/kubernetes/kubernetes/pull/85962)